.gitignore

```
@@ -0,0 +1,4 @@
+**pycache**
```

Ignore folders named `__pycache__`
```suggestion
__pycache__/
```